### PR TITLE
added the space between text and table border

### PR DESCRIPTION
**Pull request by yasirunilan**
Pull Request for Issue #14839 .

### Summary of Changes
Added the space between table border and text displayed.Added padding

### Testing Instructions
Go to Menu, Extensions -> Manage -> Discover.

### Documentation Changes Required
None

### CODE REVIEW

**Comment by yasirunilan** [comment]
I added some padding for this. Is it okay or should I create a css class and include it instead of inline styling. Any feedback?

File: administrator/components/com_installer/views/discover/tmpl/default.php [hunk 1]
```diff
@@ -115,7 +115,7 @@
 				</tbody>
 			</table>
 			<?php endif; ?>
-			<div><?php echo JText::_('COM_INSTALLER_MSG_DISCOVER_DESCRIPTION'); ?></div>
+			<div style="padding-left: 10px"><?php echo JText::_('COM_INSTALLER_MSG_DISCOVER_DESCRIPTION'); ?></div>
```
Comment (yvesh) [inline administrator/components/com_installer/views/discover/tmpl/default.php]: please never do inline styles :-) Do it in the scss / css file.

@C-Lodder can you look into that? this needs probably a different fix

**Comment by yvesh** [comment]
This needs to be changed in the scss file, btw. you are doing that against staging (3.7) and not 4.0 branch :-)

**Comment by C-Lodder** [comment]
We don't do inline styling at all. If you make any styling changes, they MUST go in the SCSS file. Bare in mind the white box also needs some padding. 

You'll then need Ruby and NodeJS to lint and compile everything back into CSS. Please read the documentation on how to set everything up properly.

If you want me to do it instead, I'll submit a PR tomorrow.

File: administrator/components/com_installer/views/discover/tmpl/default.php (hunk 1 above)
Comment (dgt41) [inline administrator/components/com_installer/views/discover/tmpl/default.php]: just wrap the text with a `<p>` should do the trick here

**Comment by yasirunilan** [comment]
Thanks for the tips, I'll give it another try. Which scss file do I need to add this styling change?

**Comment by C-Lodder** [comment]
@yasirunilan - try using a `<p>` tag as suggested by Dimitris:

```
<p><?php echo JText::_('COM_INSTALLER_MSG_DISCOVER_DESCRIPTION'); ?></p>
```

**Comment by yasirunilan** [comment]
@C-Lodder I tried it also. Left padding issue is still there.

**Comment by C-Lodder** [comment]
Ok let me look into it a little later

**Comment by yasirunilan** [comment]
okay @C-Lodder

**Comment by ciar4n** [comment]
@yasirunilan I slightly easier solution would be to add ``class="p-3"`` to the following div.. https://github.com/joomla/joomla-cms/blob/staging/administrator/components/com_installer/views/discover/tmpl/default.php#L118

**Comment by yasirunilan** [comment]
@ciar4n  thanks. I made that change and made a PR.

**Comment by yasirunilan** [comment]
I made another PR as I had done this PR into staging branch 3.7.

**Comment by RonakParmar** [comment]
We have another PR #14862 for this issue. So Can anyone close this PR? Thank you.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14842">issues.joomla.org/tracker/joomla-cms/14842</a>.</sub>

**Comment by brianteeman** [comment]
Closed as requested

**Pull request closed without merging**